src/server/api/user.js

```diff
@@ -250,119 +373,21 @@ export const resolvers = {
-      }
-
-      if (getConfig("FILTER_DUEBY", null, { truthy: 1 })) {
-        query = query.where("campaign.due_by", ">", new Date());
-      }
-
-      if (withOutCounts) {
-        return await query.select(fields);
-      } else {
-        query
-          .leftOuterJoin("campaign_contact", function() {
-            this.on("campaign_contact.assignment_id", "assignment.id").andOn(
-              "campaign_contact.is_opted_out",
-              r.knex.raw("?", [false])
-            );
-            // https://github.com/knex/knex/issues/1003#issuecomment-287302118
-          })
-          .groupBy(
-            ...fields,
-            "campaign_contact.timezone_offset",
-            "campaign_contact.message_status"
-          )
-          .select(
-            ...fields,
-            "campaign_contact.timezone_offset",
-            "campaign_contact.message_status",
-            r.knexReadOnly.raw(
-              "SUM(CASE WHEN campaign_contact.id IS NOT NULL THEN 1 ELSE 0 END) as tz_status_count"
-            )
-          );
-        const result = await query;
-        const campaignIds = result
-          .filter(a => a.use_dynamic_assignment)
-          .map(a => a.campaign_id);
-        const campaignsWithUnassigned = {};
-        let hasUnassigned = [];
-        if (campaignIds.length) {
-          hasUnassigned = await r
-            .knex("campaign_contact")
-            .where({
-              is_opted_out: false,
-              message_status: "needsMessage"
-            })
-            .whereNull("assignment_id")
-            .whereIn("campaign_id", campaignIds)
-            .select("campaign_id")
-            .groupBy("campaign_id")
-            .havingRaw("count(1) > 0");
-          hasUnassigned.forEach(c => {
-            campaignsWithUnassigned[c.campaign_id] = 1;
-          });
-        }
-        const assignments = {};
-        for (const assn of result) {
-          if (!assignments[assn.id]) {
-            assignments[assn.id] = {
-              ...assn,
-              tzStatusCounts: {}
-            };
-            if (assn.use_dynamic_assignment && campaignIds.length) {
-              assignments[assn.id].hasUnassigned =
-                campaignsWithUnassigned[assn.campaign_id] || 0;
-            }
-          }
-          if (!assignments[assn.id].tzStatusCounts[assn.message_status]) {
-            assignments[assn.id].tzStatusCounts[assn.message_status] = [];
-          }
-          assignments[assn.id].tzStatusCounts[assn.message_status].push({
-            tz: assn.timezone_offset,
-            count: assn.tz_status_count
-          });
-        }
-        return Object.values(assignments);
+    todos: async (
+      user,
+      { organizationId, withOutCounts },
+      { user: loggedInUser }
+    ) => {
+      const assignments = await getUserTodos(
+        user,
+        { organizationId, withOutCounts },
+        { user: loggedInUser }
+      );
+      if (user.id === loggedInUser.id) {
```

Comment:
if we call this, we should clear any notifications, since the user is getting the newest data.